views/entity.html

```diff
@@ -96,13 +97,11 @@ <h1 class="govuk-heading-xl">Tell us about the overseas entity</h1>
             text: "Yes",
             conditional: {
                html: entityPublicRegisterHtml
-            },
-            checked: is_on_sanctions_list === 1
+            }
```

Comment:
Hi Matthew, you should not remove the `checked` field, but you should update the controller. You can find example in other fields

Reply:
Put in a pr for the sdk: https://github.com/companieshouse/api-sdk-node/pull/384

Added checked fields to two templates as this was based on bo other and that was also missing them controller updated as well.